mlflow/server/js/src/experiment-tracking/components/runs-charts/hooks/useSampledMetricHistory.tsx

```diff
@@ -110,7 +110,7 @@ export const useSampledMetricHistory = (params: {
 
     // Base fetching function, used for both initial call and subsequent auto-refresh calls
     const fetchMetricsFn = async (isAutoRefreshing = false) => {
-      const runUuids = runUuidsSerialized.split(',').filter((runUuid: string) => runUuid !== '');
+      const runUuids = runUuidsSerialized.split(',');
```

Comment:
ahh this again, can we revert this line so it will have `.filter()` again?
I'll fill a PR targeting universe as well